ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -337,6 +337,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-private-workers-marketimage-p2-f14
```

Comment:
I think we drop the following `azure-ipi-private-p2-f14` job now, using this one to replace that.